### DISCUSSION TITLE
Default closed dropdown view in trace details

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#3367] [Ideas]
### Describe the feature or potential improvement

Ability to have a closed-by-default view of observations in trace details.

<img width="346" alt="Screenshot 2024-09-16 at 14 45 37" src="https://github.com/user-attachments/assets/1b787371-25f1-44f7-a5ca-e986bf674abc">

**Comment by marcklingen** [comment]
thanks for raising this!
